### PR TITLE
COL-654 Additional logging for poller and thumbnail generator

### PR DESCRIPTION
**Pull request by paulkerschen**
https://jira.ets.berkeley.edu/jira/browse/COL-654

These additions will give us noisier logs, but also (one hopes) a better sense of what points in the process are failing.

### CODE REVIEW

File: node_modules/col-whiteboards/lib/api.js [hunk 1]
```diff
@@ -1820,11 +1822,16 @@ var generateThumbnail = function(whiteboard, callback) {
...
       return callback(err);
     }
-    log.debug({'whiteboard': whiteboard.id}, 'Got the PNG data for a whiteboard');
+    log.info({'whiteboard': whiteboard.id}, 'Got the PNG data for a whiteboard');
 
     // Store the large image URL
     var update = {'imageUrl': canvasItem.url};
     updateWhiteboardPreview(whiteboard, update, function(err, updatedWhiteboard) {
+      if (err) {
+        log.error({'err': err}, 'Unable to update whiteboard preview');
```
Comment (johncrossman) [inline node_modules/col-whiteboards/lib/api.js]: @paulkerschen would `whiteboard.id` be useful above?

File: node_modules/col-whiteboards/lib/api.js [hunk 2]
```diff
@@ -2001,6 +2008,7 @@ var scheduleThumbnails = function() {
     Collabosphere.updateLastWhiteboardThumbnails();
 
     // Schedule the next run
+    log.info({'course': course}, 'Will run whiteboard thumbnail generator in %d seconds', THUMBNAIL_INTERVAL);
```
Comment (johncrossman) [inline node_modules/col-whiteboards/lib/api.js]: @paulkerschen ...same question: `whiteboard.id`?

File: node_modules/col-whiteboards/lib/api.js (hunk 1 above)
Comment (paulkerschen) [inline node_modules/col-whiteboards/lib/api.js]: @johncrossman Yes, will amend.

File: node_modules/col-whiteboards/lib/api.js (hunk 2 above)
Comment (paulkerschen) [inline node_modules/col-whiteboards/lib/api.js]: @johncrossman There's no specific `whiteboard.id` here - the generator runs on all whiteboards that have been scheduled.

Comment (paulkerschen) [inline node_modules/col-whiteboards/lib/api.js]: @johncrossman And in fact there's no `course` either. Must have been a bad copy/paste; I'll remove that.

**Pull request merged**